### ISSUE TITLE
Privacy Considerations

### ISSUE THREAD

**Issue by wip-abramson** [EarthProgram/did-earth#10]

**Comment by wip-abramson** [comment]
I turned this into an issue, as I figure it will be easier for us to discuss exactly what should go in this section

**Comment by wip-abramson** [comment]
For privacy the text has a lot already, just not currently in this section. May just mean a refactor?

- need to include something about linkedResources. 
- Perhaps also a reiteration that PII should not go into DIDDocs although this is in DIDCore already

**Comment by lohanspies** [comment]
@wip-abramson you are correct in stating that the privacy consideration section needs a refactor and also bring in linkedResources.

Will you be able to take a stab at it?

**Comment by wip-abramson** [comment]
Yep, I can take this @lohanspies
